ToolBox/Program.cs

```diff
@@ -302,5 +307,71 @@ private static string GetParameterOrDefault(IReadOnlyDictionary<string, object>
 
             return value.ToString();
         }
+
+        /// <summary>
+        /// Loads values from JSON text file
+        /// </summary>
+        private static Dictionary<string, object> LoadOptionsObjectFromConfigurationFile(string path)
```

Comment:
Perhaps this logic makes more sense being hidden within the `ParseArguments` method instead of living in `main` ?

Reply:
has been moved to ApplicationParser